tests/play_game_test.py

```diff
@@ -30,3 +30,10 @@ def test_valid_input(self):
         play_game = PlayGame(new_board, ui)
         assert play_game.valid_input("1") == True
         assert play_game.valid_input("0") == False
+
+    @pytest.mark.parametrize("input, expected", [("O", "X"), ("X", "O")])
+    def test_next_player(self, input, expected):
```

Comment:
Tests tell a story about the code base, could the test name be a little more descriptive about what you are testing here?

Reply:
Just want to make sure if I understand
 this correctly.This test is testing next_player method, so should I called test_next_player_method or do you mean more like a story, test_if_players_alternate_turns

Reply:
Ok hmm after reading the actual next_player method, I would suggest something like "switch_players" for the method name in the play_game class that way it's a little more straightforward what the method does. Then the test could follow the same naming convention.